app/produce-steps-json.py

```diff
@@ -154,24 +154,28 @@ def main(region,filter,destination,impregion,writetput,readtput, spikedread, s3l
         autoscale_min_reset_read_capacity = None
         tableS3Path = s3ExportPath + "/" + table['name']
 
-        # Does this table have autoscaling enabled?
-        scalable_target_info = scalable_target_exists(region,"table/" + table['name'],"dynamodb:table:ReadCapacityUnits")
-        if scalable_target_info is not None:
+        # Check if table is set to On Demand Capacity
+        if int(table['read']) > 0:
```

Comment:
Might be good to have an else that logs that it's ondemand capacity and that it will not be spiked.